packages/paste-website/src/pages/components/modal/index.mdx

```diff
@@ -300,12 +297,14 @@ By using both alignment types, you are able to create directional alignments of
 
 ### When to use Modal
 
-Use Modals to
+Use a Modal to:
 
 - Request the user enter information to continue the current process.
 - Guide the user through a complex workflow with a series of steps, while still maintaining context of the task that initiated the modal.
 
-If you need to show an important warning to prevent or correct critical errors, use an [Alert Dialog](/components/alert-dialog) instead.
+Do not use modals to show important warnings, since they can be dismissed without the user acknowledging the information. Instead, use an [Alert Dialog](/components/alert-dialog) instead.
```

Comment:
```suggestion
Do not use modals to show important warnings, since they can be dismissed without the user acknowledging the information. Instead, use an [Alert Dialog](/components/alert-dialog).
```